cluster/src/main/java/com/hartwig/pipeline/calling/structural/StructuralCaller.java

```diff
@@ -82,7 +83,8 @@ public StructuralCallerOutput run(final SomaticRunMetadata metadata, final Align
         InputDownload referenceBam = new InputDownload(pair.reference().finalBamLocation());
         InputDownload referenceBai = new InputDownload(pair.reference().finalBaiLocation());
 
-        bash.addCommands(asList(tumorBam, referenceBam, tumorBai, referenceBai, referenceGenomeDownload, gridssConfigFiles, gridssPonFiles));
+        bash.addCommand(new BatchInputDownload(referenceBam, referenceBai, tumorBam, tumorBai));
```

Comment:
Yeah it was an optimisation that I think is applicable across the board. Kinda got lumped in with the grab-bag of GRIDSS stuff because it was already in my workspace but yes it should have been done separately.